hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -90,22 +91,45 @@ object HoodieSparkUtils {
     new InMemoryFileIndex(sparkSession, globbedPaths, Map(), Option.empty, fileStatusCache)
   }
 
-  def createRdd(df: DataFrame, structName: String, recordNamespace: String): RDD[GenericRecord] = {
+  def createRdd(df: DataFrame, structName: String, recordNamespace: String, mayBeUseLatestTableSchema: Boolean): RDD[GenericRecord] = {
```

Comment:
I haven't fixed all callers to set the last arg from config. for eg: SparkParquetBootstrapDataProvider and SourceFormatAdaptor calls with false as last arg. not sure if its required and if yes, we can take it as a follow up and not bloat this PR.

Reply:
Can you file a ticket for this ?

Reply:
https://issues.apache.org/jira/browse/HUDI-1907